src/index.html

```diff
@@ -29,6 +29,6 @@
 </head>
 
 <body>
-    <div id="app"></div>
+    <ct-application />
```

Comment:
```suggestion
    <ct-application></ct-application>
```

custom elements can't be self-closing https://stackoverflow.com/questions/23961178/do-custom-elements-require-a-close-tag